### ISSUE TITLE
ValueError: Got tab_id None when downloading page with SSL errors

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/qutebrowser#1413]
When doing `:download https://expired.identrustssl.com/`:

```
14:49:40 DEBUG    webview    networkmanager:on_ssl_errors:199 SSL errors [<qutebrowser.browser.network.networkmanager.SslError error='UnableToGetLocalIssuerCertificate' string='The issuer certificate of a locally looked up certificate could not be found'>, <qutebrowser.browser.network.networkmanager.SslError error='UnableToVerifyFirstCertificate' string='No certificates could be verified'>], strict ask
14:49:40 ERROR    misc       crashsignal:exception_hook:211 Uncaught exception
Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/browser/network/networkmanager.py", line 222, in on_ssl_errors
    mode=usertypes.PromptMode.yesno, owner=reply)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/browser/network/networkmanager.py", line 170, in _ask
    tab=self._tab_id)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/utils/objreg.py", line 215, in get
    reg = _get_registry(scope, window, tab)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/utils/objreg.py", line 202, in _get_registry
    return _get_tab_registry(window, tab)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/utils/objreg.py", line 147, in _get_tab_registry
    raise ValueError("Got tab_id None (win_id {})".format(win_id))
ValueError: Got tab_id None (win_id 0)
```

**Comment by Kingdread** [comment]
I guess this can be closed as it should be fixed by #1473

**Comment by The-Compiler** [comment]
Indeed! If you mention "Fixes #1413" in the commit or PR message that'll happen automatically, by the way :wink:
